cypress/support/step_definitions/html.js

```diff
@@ -41,7 +41,7 @@ Then('I set on {string} text {string}', (templateSelector, templateValue) => {
   const selector = nunjucks.renderString(templateSelector, cy.context);
   const value = nunjucks.renderString(templateValue, cy.context);
 
-  cy.get(selector).type(value);
+  cy.get(selector).clear().type(value);
```

Comment:
why `.clear()` ?

Reply:
to clear input before typing text, make a functionnal test bug with the branch filter